swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/annotations/RequestParamColFmt.java

```diff
@@ -0,0 +1,24 @@
+package io.servicecomb.swagger.extend.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.core.annotation.AliasFor;
+
+@Target({ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface RequestParamColFmt {
```

Comment:
do  not  create a new annotation.
use  swagger annotation

Reply:
use swagger annotation is diffcult to solve this problem .beacause 1: queryparameter has fixed default collection value ,it means should extend it to cover this method to return collection value.2:
no matter i choose ApiParam or ApiImplicitParam ,ApiParam index cannot be fixed ,so cannot process, ApiImplicitParam dataType has no queryType,so cannot process

Reply:
we have discuss this weeks age, and i have gave you the solution......